c/include/libsbp/observation.h

```diff
@@ -612,6 +612,21 @@ coordinate system
 } msg_almanac_glo_t;
 
 
+/** GLONASS SV orbital and frequency slots mapping information
+ *
+ * The message reports mapping information regrding GLONASS SV orbital and
+ * frequency slots. Frequency numbers -7 to 6 have been reported as 1 to 14
+ * correspondingly. 0xFF for invalid. The 0th element and elements from 29th
+ * to 30th of the map are always invalid.
+ */
+#define SBP_MSG_FCNS_GLO             0x0072
+typedef struct __attribute__((packed)) {
+  u16 wn;      /**< GPS Week number [week] */
+  double tow;     /**< GPS Time of week [s] */
```

Comment:
It looks like `u32` is a more typical type for `tow` in SBP messages and units are in `[ms]`. Would it make sense to change `double tow_s` to `u32 tow_ms` then?
If time is unknown, then we could use `(u32)TOW_INVALID` and describe it explicitly in the comments. We may or may not use it, but at least it would give us more freedom to send it even if the time is really unknown.